Cubical/Foundations/Pointed/Properties.agda

```diff
@@ -13,3 +20,48 @@ open import Cubical.Data.Prod
 
 _×∙_ : ∀ {ℓ ℓ'} (A∙ : Pointed ℓ) (B∙ : Pointed ℓ') → Pointed (ℓ-max ℓ ℓ')
 A∙ ×∙ B∙ = ((typ A∙) × (typ B∙)) , (pt A∙ , pt B∙)
+
+{- composition of pointed maps -}
+_∘*_ : ∀ {ℓA ℓB ℓC} {A : Pointed ℓA} {B : Pointed ℓB} {C : Pointed ℓC}
```

Comment:
It would be better if all definitions related to pointed things used the same symbol, so please change `*` to `∙` everywhere

Reply:
Oops, this should have been part of the review

Reply:
I stuck to Foundations/Pointed/Base, where for pointed functions the asterisk is used, but I can change that aswell.